### ISSUE TITLE
Remove bookmark list

### ISSUE THREAD

**Issue by LukeMauldin** [LintaoAmons/bookmarks.nvim#56]
I have the following bookmark lists created:
foo
bar

How do I delete the "bar" bookmark list?

**Comment by LintaoAmons** [comment]
Method 1:
open `BookmarkList` then `<c-d>` in the telescope picker
Method 2:
open `BookmarkTree` then `D` the list node

FYI: you can use `BookmarkInfo` to check the available actions of `BookmarkTree`

```lua
treeview = {
  keymap = {
    copy = "c",
    create_list = "a",
    cut = "x",
    delete = "D",
    ["goto"] = "g",
    level_up = "u",
    move_down = "<localleader>j",
    move_up = "<localleader>k",
    paste = "p",
    quit = { "q", "<ESC>" },
    refresh = "R",
    rename = "r",
    reverse = "t",
    set_active = "m",
    set_root = ".",
    show_info = "i",
    toggle = "o"
  },
}
```

**Comment by LukeMauldin** [comment]
Thank you for the clarification.  Could be good to clarify in the docs because it is a question that I think most people new to the package could have and wasn't immediately obvious to me.
Does this package integrate with fzf-lua at all or would I have to use telescope?

**Comment by LintaoAmons** [comment]
Yeah, I updated the README file. I hope it makes it more obvious.
Currently don't have extra effort to support fzf-lua yet, but it should be able to extend in the future, since all we need is write a fzf-lua picker

**Comment by LukeMauldin** [comment]
Is there an open source example of a fzf-lua picker that I could reference
and maybe take a stab at the implementation?  I am relatively new to neovim
but I am liking the community.

**Comment by LintaoAmons** [comment]
I think fzf-lua provides quite a lot of information about how to make a customised picker in their documentation 

And what you need to do is provide fzf-lua version of `lua/bookmarks/picker/bookmark-picker.lua ` etc.

I will make this as done since you already know how to remove a list
